common/v2/components/AssetIcon.tsx

```diff
@@ -22,13 +24,14 @@ const SImg = styled('img')`
 `;
 
 interface Props {
-  symbol: TSymbol;
+  uuid: string;
   size?: string;
   className?: string;
 }
 
-function AssetIcon({ symbol, size = '32px', className }: Props) {
-  const iconUrl = getIconUrl(symbol);
+function AssetIcon({ uuid, size = '32px', className }: Props) {
+  const { assetMapping } = useContext(RatesContext);
```

Comment:
After my PR (https://github.com/MyCryptoHQ/MyCrypto/pull/3230) for fetching assets info from API is merged it will be super simple getting coingecko directly from asset and assetMappimg is not needed anymore.

Reply:
Im not going to touch this one then since @amacar is handling c: